yarn.lock

```diff
@@ -12526,12 +13401,10 @@ strip-bom@^3.0.0:
   resolved "https://registry.yarnpkg.com/strip-bom/-/strip-bom-3.0.0.tgz#2334c18e9c759f7bdd56fdef7e9ae3d588e68ed3"
   integrity sha1-IzTBjpx1n3vdVv3vfprj1YjmjtM=
 
-strip-comments@^1.0.2:
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/strip-comments/-/strip-comments-1.0.2.tgz#82b9c45e7f05873bee53f37168af930aa368679d"
-  dependencies:
-    babel-extract-comments "^1.0.0"
-    babel-plugin-transform-object-rest-spread "^6.26.0"
+strip-comments@^2.0.1:
+  version "2.0.1"
```

Comment:
Retrigger the deploy may not help, check out the deploy log

> 11:03:41 AM: error strip-comments@2.0.1: The engine "node" is incompatible with this module. Expected version ">=10".

Reply:
Thanks, I just notice it, it's a [bug](https://github.com/GoogleChrome/workbox/issues/2415)  related to workbox5.1.1